styles/style.css

```diff
@@ -0,0 +1,191 @@
+.container {
+    display: grid;
+    border: solid black 2px;
+    flex-wrap: wrap;
+}
+
+h1 {
+    font-family: 'Tangerine', cursive;
+}
+
+ul {
+    list-style-type: none;
+}
+
+nav ul li:hover {
+    text-decoration: underline;
+    color: red;
+}
+
+nav {
+    flex-wrap: wrap;
+}
+
+#first-border {
+    border-top: 2px solid grey;
+}
+
+.border {
+    border-bottom: 2px solid grey;
+}
+
+.navigation {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr 1fr;
+    font-weight: bold;
+    flex-wrap: wrap;
+}
+
+nav ul li:nth-child(2) {
+    grid-column: 3 / 4;
+}
+
+#main {
+    flex-wrap: wrap;
+}
+
+#startrly {
+    padding-top: 70px;
+    display: grid;
+    grid-template-rows: 1fr 1fr 1fr 1fr;
+    /* display: flex; */
+    text-align: center;
+    justify-content: space-around;
+    font-size: 20px;
+}
+
+#action-call {
+    border: solid gray 2px;
+    border-radius: 5px;
+    margin: 25px;
+    padding: 20px;
+    font-size: 15px;
+    text-decoration: none;
+    background-color: rgb(50, 228, 65);
+}
+
+.quote {
+    /* display: flex; */
+    text-align: center;
+    font-style: italic;
+    /* justify-content: space-around; */
+    /* border-top: 1px solid lightgrey;
+    border-bottom: 1px solid lightgrey; */
+    margin-bottom: 3em;
+}
+
+.quote>blockquote {
+    font-style: italic;
+    padding: 5em 20em 1em 20em;
+    font-size: 1em;
+}
+
+.quote>p {
+    font-style: italic;
+    text-align: right;
+    padding-right: 20em;
+}
+
+blockquote:before {
+    color: #000;
+    content: open-quote;
+    font-size: 4em;
+    line-height: 0.1em;
+    margin-right: 0.25em;
+    vertical-align: -0.4em;
+}
+
+blockquote:after {
+    color: #000;
+    content: close-quote;
+    font-size: 4em;
+    line-height: 0.1em;
+    margin-right: 1em;
+    vertical-align: -0.5em;
+}
+
+.from {
+    text-align: right;
+    font-style: italic;
+    padding-right: 250px;
+}
+
+.heading {
+    padding-top: 50px;
+    text-align: center;
+}
+
+.why {
+    display: flex;
+    flex-direction: row;
+    list-style: none;
+    justify-content: space-around;
+    align-items: flex-start;
+    overflow: visible;
+}
+
+.why>li>img {
+    float: left;
+    margin: 4px;
+}
+
+.why>li>p {
+    margin-top: 0;
+    flex-wrap: wrap;
+    width: 278px;
+}
+
+.circle {
+    border-radius: 50%;
+}
+
+.first-three {
+    display: flex;
+    justify-content: space-evenly;
+    flex-wrap: wrap;
+}
+
+.last-two {
+    display: flex;
+    justify-content: space-evenly;
+    flex-wrap: wrap;
+}
```

Comment:
i think first-three and last-two could be combined into just one flex with the command flex-wrap to achieve the same effect